check-docstyle.sh

```diff
@@ -3,14 +3,30 @@ separate_files="setup.py"
 pass=0
 fail=0
 
+
+# set up terminal colors
+NORMAL=$(tput sgr0)
+RED=$(tput bold && tput setaf 1)
+GREEN=$(tput bold && tput setaf 2)
+YELLOW=$(tput bold && tput setaf 3)
+
 function prepare_venv() {
-    VIRTUALENV=`which virtualenv`
-    if [ $? -eq 1 ]; then
-        # python34 which is in CentOS does not have virtualenv binary
-        VIRTUALENV=`which virtualenv-3`
-    fi
+		# we want tests to run on python3.6
+	printf 'checking alias `python3.6` ... '
+	PYTHON=$(which python3.6 2> /dev/null)
+	if [ -z ${PYTHON} ]; then
```

Comment:
I'd suggest to use the exit code from `which` command. Please see `man which` and the `$?` variable.

Reply:
Thanks for the suggestion, no problem to change it. Could you please elaborate on the reason why is it better?

Reply:
this is common idiom for many tools to communicate via exit code, so you don't have to use $() and/or to parse output. For example:

```
$ which -X 2> /dev/null
Usage: /usr/bin/which [-a] args

$ echo $?
2
```

Ie the std. output still contains some garbage that will be stored in $PYTHON in your code (so it might seem we found the executable), but the exit code clearly states what happened.

But it is really a nitpick in this particular script :)

Reply:
Okay, cool .. changing :+1: